plugins/hydra_ray_launcher/examples/upload_download/conf/hydra/launcher/custom_ray_aws.yaml

```diff
@@ -9,12 +9,11 @@ sync_up:
   # we leave target_dir to be null
   # as a result the files will be synced to a temp dir on remote cluster.
   # the temp dir will be cleaned up after the jobs are done.
-  # recommend to leave target_dir to be null if you are syncing code/artifacts to remote cluster so you don't need
+  # recommend to leave target_dir to be null if you are syncing code/artifacts
+  # to remote cluster so you don't need
   # configure $PYTHONPATH on remote cluster
-  include: ['model', '*.py']
```

Comment:
could you provide more info on why `include`  is removed?

Reply:
It's because ray doesn't define it. They only have `rsync_exclude` and `rsync_filter` in the [schema](https://github.com/ray-project/ray/blob/35ec91c4e04c67adc7123aa8461cf50923a316b4/python/ray/autoscaler/ray-schema.json#L286-L293), which is a bummer :(. So I was thinking that either we keep the current rsync_up/down in `_launcher_util` and file a PR in ray for `--include` support or we only use `exclude` and assume that anything that isn't excluded or filtered using `rsync_filter` is going to be downloaded (which is what I'm assuming in the PR).

Reply:
I see! Thank you!
I have created an issue with ray here https://github.com/ray-project/ray/issues/15859, let's see if they will consider adding this option.
For now, I suggest we keep the rsync down/up configuration.